runtime/frequency/src/lib.rs

```diff
@@ -615,12 +618,43 @@ impl pallet_msa::Config for Runtime {
 parameter_types! {
 	/// The maximum number of eras over which one can claim rewards
 	pub const ProviderBoostHistoryLimit : u32 = 30;
-	/// The number of chunks of Reward Pool history we expect to store
+	/// The number of Reward Pool history chunks we expect to store
 	pub const RewardPoolChunkLength: u32 = 5;
+	/// Max difference between PTE and current block number
+	pub const MaxPteDifferenceFromCurrentBlock: u32 = 60 * DAYS;
 }
 // RewardPoolChunkLength MUST be a divisor of ProviderBoostHistoryLimit
 const_assert!(ProviderBoostHistoryLimit::get() % RewardPoolChunkLength::get() == 0);
 
+/// Configuration definitions for Staking types
+pub struct FrequencyStakingConfigProvider<Runtime> {
+	_marker: PhantomData<Runtime>,
+}
+impl<T: frame_system::Config> StakingConfigProvider<T> for FrequencyStakingConfigProvider<T> {
+	fn get(staking_type: StakingType) -> StakingConfig<T> {
+		match staking_type {
+			StakingType::CommittedBoost =>
+				StakingConfig::<T> {
+					// TODO: TBD by leadership
+					reward_percent_cap: Permill::from_parts(8_000),
```

Comment:
noting here this is 20.8% per year cap

Reply:
Placeholder; still waiting for official values